BritneyS/GetFed
# APIClient api call successful parsing

## What you did :question:
- Rewrote APIClient for successful parsing

## How you did it :white_check_mark:
- Removed URL handing from APIClient class
- Manually decoded Nutrients properties, since 'fat' and 'protein' were not guaranteed keys for every food result
- Made request and loaded parsed data into 'searchResults' property asynchronously


## How to test it :microscope:
- Where `makeRequest` is called in viewDidLoad, replace the `[redacted]` url string with a valid request url
- Run the app
- Tap 'Food Search' button
- Note that the parsed data is printed onto the console


## Screenshots (if applicable) :camera:
![screen shot 2018-11-20 at 3 17 50 pm](https://user-images.githubusercontent.com/8409475/48800895-08882a80-ecd9-11e8-8344-495f88776d58.png)
